x-pack/plugins/security_solution/common/endpoint/schema/metadata.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema, TypeOf } from '@kbn/config-schema';
+import { HostStatus } from '../types';
+
+export const GetMetadataListRequestSchemaV2 = {
+  query: schema.object(
```

Comment:
so the default value below actually makes this optional. you can confirm this in the added test:
```
    it('should correctly use default values', () => {
      const expected = { page: 0, pageSize: 10 };
      expect(query.validate(undefined)).toEqual(expected);
      expect(query.validate({ page: undefined })).toEqual(expected);
      expect(query.validate({ pageSize: undefined })).toEqual(expected);
      expect(query.validate({ page: undefined, pageSize: undefined })).toEqual(expected);
    });
```

interestingly, wrapping in `schema.maybe` prevented the default values from working.

Reply:
Does `.maybe()` have a default value?

Reply:
no, single arg only.

Reply:
After our discussion - this is working as desired. the metadata API does not need any query params to work - it can be called with just `GET .../metadata` and it will work by applying defaults.